scripts/bootstrap-plugin

```diff
@@ -21,8 +21,6 @@ if [ -d $pluginname ]; then
   err "There is already a plugin with that name. Cowardly refusing to overwrite it."
 fi
 
-buildConfigFilename=${pluginname}/build.config.js
-
 mvn archetype:generate -DarchetypeGroupId=org.graylog -DarchetypeArtifactId=graylog-plugin-archetype -DartifactId=$pluginname
 
-sed 's,\(../graylog2-server/graylog2-web-interface\),../\1/,' < ${buildConfigFilename}.sample > ${buildConfigFilename}
+echo ${pluginname}/ >> ${prefix}/.gitignore
```

Comment:
Not a bad idea, but this would require more work (changing the plugin's `build.conf.js`, changing `webpack.combined.config.js` of our web interface, etc.), which is not too difficult, but introduces more things which could go wrong. OTOH having the directory in `.gitignore` multiple times (what happens only when a plugin dir is created, removed and recreated with the exact same name) wouldn't harm anybody.